app/uk/gov/hmrc/childcarecalculatorfrontend/navigation/MaximumHoursNavigator.scala

```diff
@@ -19,6 +19,8 @@ package uk.gov.hmrc.childcarecalculatorfrontend.navigation
 import javax.inject.Inject
 import play.api.mvc.Call
 import uk.gov.hmrc.childcarecalculatorfrontend.controllers.routes
+import uk.gov.hmrc.childcarecalculatorfrontend.controllers.benefits.routes.DoYouGetCarersAllowanceController
+import uk.gov.hmrc.childcarecalculatorfrontend.controllers.benefits.routes.DoesPartnerGetCarersAllowanceController
```

Comment:
You can do this with
import uk.gov.hmrc.childcarecalculatorfrontend.controllers.benefits.{routes => benefitsRoutes}
That way noone will need to fiddle with imports for other pages as benefitsRoutes can just be referenced